src/main/java/org/myrobotlab/service/Runtime.java

```diff
@@ -3616,13 +3567,14 @@ public static Double getBatteryLevel() {
       } else if (platform.isLinux()) {
         // TODO This is incorrect, will not work when unplugged
         // and acpitool output is different than expected,
-        // at least on Ubuntu 22.04
-        String ret = Runtime.execute("acpitool");
-        int pos0 = ret.indexOf("Charging, ");
+        // at least on Ubuntu 22.04 - consider oshi library

```

Comment:
oshi is already pulled in as a transient dependency of some other service, in a different branch I added it as a Runtime dep and it worked great

Reply:
Ya, makes sense ...  I've been looking at that for a while